code/addons/docs/src/preset.ts

```diff
@@ -153,6 +160,10 @@ const docs = (docsOptions: DocsOptions) => {
   };
 };
 
+export const addons: StorybookConfig['addons'] = [
+  dirname(require.resolve('@storybook/react-dom-shim/package.json')),
```

Comment:
This preset will be added twice, if the user is using docs AND react-renderer, right?

Reply:
See: https://github.com/storybookjs/storybook/pull/21197/files#r1115437543

I'm guessing if the path is exactly the same, 1 will be skipped (might log a message about that in the command-line).

Reply:
It seems to be OK. It runs it twice, I think (which isn't really ideal) but it's idempotent so I guess it doesn't really matter. But ideally our preset system would handle that.